From 491520f858c03d6d5338420fbcbff780dd483dee Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Fri, 4 Mar 2022 10:07:20 -0500
Subject: [PATCH 75/80] Fix Play Services crashing on Galaxy S21 Ultra

Samsung defines the presence of UWB framework in /vendor
This is wrong, because the UWB framework is hosted in /system
Which means that when booting a GSI, framework will say it has UWB
while it doesn't
---
 rw-system.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/rw-system.sh b/rw-system.sh
index 62ded07..1f272fe 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -1105,4 +1105,6 @@ if getprop ro.vendor.build.fingerprint | grep -iq -e motorola/liber; then
   mount -o bind /mnt/phh/audio_policy_configuration.xml /vendor/etc/audio_policy_configuration.xml
   chcon -h u:object_r:vendor_configs_file:s0 /vendor/etc/audio_policy_configuration.xml
   chmod 644 /vendor/etc/audio_policy_configuration.xml
-fi
\ No newline at end of file
+fi
+
+mount /system/phh/empty /vendor/etc/permissions/samsung.hardware.uwb.xml
-- 
2.25.1

